src/libraries/System.ServiceProcess.ServiceController/src/System/ServiceProcess/ServiceBase.cs

```diff
@@ -25,17 +26,17 @@ public class ServiceBase : Component
     {
         private SERVICE_STATUS _status;
         private IntPtr _statusHandle;
-        private ServiceControlCallbackEx _commandCallbackEx;
-        private ServiceMainCallback _mainCallback;
-        private ManualResetEvent _startCompletedSignal;
-        private ExceptionDispatchInfo _startFailedException;
+        private ServiceControlCallbackEx? _commandCallbackEx;
+        private ServiceMainCallback? _mainCallback;
+        private ManualResetEvent? _startCompletedSignal;
+        private ExceptionDispatchInfo? _startFailedException;
         private int _acceptedCommands;
-        private string _serviceName;
+        private string? _serviceName;
```

Comment:
This does not seem to be nullable, set in constructor with "" using ServiceName property, and ServiceName property not allowing null for set

Reply:
Good catch. Fixed.